README.md

```diff
@@ -1,5 +1,8 @@
 # COVID Moonshot Data
 
+[![License: CC0-1.0](https://licensebuttons.net/l/zero/1.0/80x15.png)](http://creativecommons.org/publicdomain/zero/1.0/)
```

Comment:
Can we say:
```
Data: [![License: CC0-1.0](https://licensebuttons.net/l/zero/1.0/80x15.png)](http://creativecommons.org/publicdomain/zero/1.0/)

Software and documentation: [![License: MIT](https://img.shields.io/badge/License-MIT-yellow.svg)](https://opensource.org/licenses/MIT)
```

Reply:
@mikemhenry : Actually, looks like software already has a license, and this should be moved below.
See https://github.com/postera-ai/COVID_moonshot_submissions/pull/85/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R83

Reply:
Maybe we can just add "This repository is still under active curation to improve the usability of data for downstream methods assessment and machine learning. Please watch this repository for future updates."

Reply:
I added that note at the bottom, we can do both. I will also add that note about it being under active curation as well